tests/asserts/final/publication/geoserver.py

```diff
@@ -10,22 +10,30 @@
 from test_tools import util as test_util, geoserver_client, process_client, assert_util
 
 
-def validate_wms_1_3_0(workspace, publ_type, name):
+def validate_wms(workspace, publ_type, name, *, version=None):
     assert publ_type == process_client.LAYER_TYPE
 
-    version = '1.3.0'
-    wms_url = wms.get_wms_url(workspace)
-    headers = {
-        settings.LAYMAN_GS_AUTHN_HTTP_HEADER_ATTRIBUTE: settings.LAYMAN_GS_USER,
-    }
-    wms_inst = gs_util.wms_proxy(wms_url, version=version, headers=headers)
+    if version:
+        wms_url = wms.get_wms_url(workspace)
+        headers = {
+            settings.LAYMAN_GS_AUTHN_HTTP_HEADER_ATTRIBUTE: settings.LAYMAN_GS_USER,
+        }
+        wms_inst = gs_util.wms_proxy(wms_url, version=version, headers=headers)
+    else:
+        wms_inst = wms.get_wms_proxy(workspace)
```

Comment:
Split asserts of internal geoserver and geoserver proxy to different modules, e.g. `geoserver` and`geoserver_proxy`. Probably encapsulate validation of WMS instance and use it in both modules.